README.md

```diff
@@ -1 +1,25 @@
-# edge-api-utils
\ No newline at end of file
+# edge-api-utils
+
+Copyright (C) 2021 The Open Library Foundation
+
+This software is distributed under the terms of the Apache License,
+Version 2.0. See the file "[LICENSE](LICENSE)" for more information.
+
+## Introduction
+The purpose of this edge API utils is the combine classes/utilities/constants shared by both modules: edge-common and edge-common-spring.
+
+## Additional information
+
```

Comment:
It probably makes sense to copy over relevant documentation from the edge-common README, e.g. sections about secure stores, caches, etc.

Reply:
Added